packages/material-ui-lab/src/TimelineItemDot/TimelineItemDot.js

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    display: 'flex',
+    alignSelf: 'baseline',
+    borderStyle: 'solid',
+    borderWidth: 2,
+    flex: 0,
+    padding: 4,
+    borderRadius: 999,
+    // marginRight: 'auto',
+    // marginLeft: 'auto',
+    top: 8,
+    position: 'relative',
+    boxShadow: theme.shadows[2],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="default"`. */
+  defaultDefault: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: theme.palette.grey[300],
+  },
+  /* Styles applied to the root element if `color="default"` and `variant="outlined"`. */
+  outlinedDefalut: {
+    borderColor: theme.palette.grey[300],
+    backgroundColor: 'white', // TODO: fix color white
```

Comment:
Is there a `palette` value that can be used here?

Reply:
```suggestion
    backgroundColor: theme.palette.background.paper,
```

Reply:
I think perhaps it should be transparent in order to show the background color. You'd need to adjust the tail though so that it isn't visible through the "hole". Perhaps with a gap between the dot and line like the Stepper has?

![image](https://user-images.githubusercontent.com/357702/83944018-6cf5d680-a7f8-11ea-8b84-c0c7a6e87c7d.png)

I also used `grey[500]`, as `300` looks kinda disabled. There might be a better palette value to use for that too.

Reply:
I added transparent background on the outline variants, and used `grey[500]`. This is how it looks like now:
![image](https://user-images.githubusercontent.com/4512430/83964055-78a1d580-a8aa-11ea-80be-056ebbf182b6.png)

I needed to add wrapper component around the `TimelineDot` and `TimelineConnector` to avoid crazy style overrides and dynamically adjusting the spacing between them.